powerform/pf-deployment-context/functions/public/initialize-powerform.ps1

```diff
@@ -0,0 +1,42 @@
+$global:_PfConfig
+
+Function Initialize-PowerForm
```

Comment:
This is now the command on how you can "initialize powerform" to get the default config values or to override those values with a custom config file.